src/Milestones/Block.php

```diff
@@ -29,6 +29,10 @@ public function addBlock() {
 						'type'    => 'string',
 						'default' => '',
 					],
+					'ids'         => [
```

Comment:
@henryholtgeerts was this renamed to `forms` somewhere?

Reply:
Initially, I had used the term "forms" for this attributes, but changed it to "ids" to better reflect what is actually being stored. Thought I've removed all references to "forms", but let me know if you see any.

Reply:
Ah, ok. I got it backwards :)